apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -444,7 +444,7 @@ defmodule OMG.Watcher.ExitProcessor.Core do
 
     invalid_piggybacks = ExitProcessor.Piggyback.get_invalid_piggybacks_events(state, known_txs_by_input)
 
-    # TODO: late piggybacks are critical, to be implemented in OMG-408
+    # TODO: late piggybacks are critical, to be implemented in https://github.com/omisego/elixir-omg/issues/671
```

Comment:
thanks for picking this up and fixing, this was misleading and put in too hastily.

If we're fixing this here, maybe even let's keep it to the #671 and ditch the `TODO` and the empty list `late_invalid_piggybacks` placeholder. At this point it became more misleading than helping. Thanks